dg-core/src/main/java/org/finra/datagenerator/consumer/EquivalenceClassTransformer.java

```diff
@@ -413,6 +413,11 @@ public void transform(DataPipe cr) {
                         break;
                 }
                 entry.setValue(b.toString());
+
+            } else if (value.contains("$")) {                      //Added by Shraddha Patel
```

Comment:
I would suggest that we use 
if (value.startsWith("$"))
instead of contains just like how we do the check for "%".

Again, My other comment is that we need a high level strategy to figure out the special notation such as '$' and '%' within a variable definition needs supporting escape characters if the user indeed intends to have a $ or % in the variable at the start as a static string.